create_db.py

```diff
@@ -0,0 +1,66 @@
+from clickhouse import ClickHouse, RepoClickHouseClient
+import argparse
+from clickhub import load_config, load_types
+import types
+
+
+parser = argparse.ArgumentParser(
+    description="github importer",
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+)
+
+parser.add_argument("-c", "--config", default="config.yml", help="config")
+parser.add_argument("-d", "--debug", action="store_true", help="debug")
+
+args = parser.parse_args()
+
+config = load_config(args.config)
+types = load_types()
+
+clickhouse = ClickHouse(
+    host=config["host"],
+    port=config["port"],
+    native_port=config["native_port"],
+    username=config["username"],
+    password=config["password"],
+    secure=config["secure"],
+)
+client = RepoClickHouseClient(clickhouse)
+
+client.query_row("CREATE DATABASE IF NOT EXISTS git")
+
+client.query_row(types[0][3])
```

Comment:
can we just loop over all 3. Also don't release on [3] for the create statement. Index by name.